koku/masu/util/common.py

```diff
@@ -204,3 +208,30 @@ def date_range_pair(start_date, end_date, step=5):
         start_date = date + timedelta(days=1)
     if len(dates) != 1 and end_date not in dates:
         yield start_date.date(), end_date.date()
+
+
+def log_date_deprecation_warning(date) -> None:
+    """Log that an object should be a date."""
+    if not isinstance(date, datetime.date):
+        LOG.warn(date
```

Comment:
These warnings are a temporary measure that I plan to remove later after things are all converted to `date`